README.md

```diff
@@ -1,17 +1,17 @@
 # gr-rswaveforms
-GNURadio sink that creates Rohde & Schwarz SMU waveform files (.wv) for R&S signal generators (e.g. SMU, SMBV).
+
+A GNU Radio module for creating Rohde & Schwarz SMU waveform files (.wv) for Rohde & Schwarz signal generators (e.g. SMU, SMV).
```

Comment:
I am not familiar with the SMV, but I know the SMBV so I don't think it was a typo per se.

Reply:
Changed SMV back to SMBV. :+1: